docs/superpowers/plans/2026-05-17-osi-bootstrap-init.md

```
@@ -0,0 +1,342 @@
+---
+
+### Task 3: Add stamp file to sysupgrade.conf
+
+**Files:**
+- Modify: `conf/full_raspberrypi_bcm27xx_bcm2712/files/etc/sysupgrade.conf`
+
+- [ ] **Step 1: Add the stamp file path**
+
+Read the current `sysupgrade.conf` and append `/etc/osi-bootstrap.done` before the final line. The file currently reads:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)